.github/workflows/build-common.yml

```diff
@@ -168,10 +168,22 @@ jobs:
       - name: Upload agent jar
         uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
         with:
-          name: opentelemetry-javaagent
+          name: opentelemetry-javaagent.jar.zip
```

Comment:
I think it would be better to call it `opentelemetry-javaagent.jar`. With your changes on the artifacts page it is nicely name `opentelemetry-javaagent.jar.zip` but when you download it you get `opentelemetry-javaagent.jar.zip.zip`. Same with the sbom zip, after download it is named `opentelemetry-java-instrumentation-SBOM.zip.zip`

Reply:
Ok, I was basing off of the text displayed.  I didn't realize it added `.zip` when it goes to download. I'll remove.